root/edit/SubscribedEdits.js

```diff
@@ -27,6 +27,13 @@ const SubscribedEdits = ({
   <Layout fullWidth title={l('Edits for Your Subscribed Entities')}>
     <div id="content">
       <h1>{l('Edits for Your Subscribed Entities')}</h1>
+
+      <p>
+        {l(`This page lists edits linked to entities you are directly
+            subscribed to, as well as edits linked to all entities which are
+            part of a collection you are subscribed to.`)}
```

Comment:
Would it be helpful to link the "collection you are subscribed to" text to `/user/foo/subscriptions/collection`?

Reply:
Hmm. Possibly? We don't pass user info here, but I guess we could, if you think it's worth it.

Reply:
Right, since we don't have user info here at the moment and I'm not sure how helpful it will be, I think we can pass on doing that for now.